_posts/2022/2022-01-31-infrastructure-software-we-use-to-run-our-saas.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Software & Infrastructure we use to Run Our SaaS"
+description: "If you do a lot with a small team, you have to pay for softare"
+layout: article
+category: ruby
+image:
+  base: "2022/infrastructure-software-we-use-to-run-our-saas"
+  alt: "Frame of a building under construction"
+  credit: "Jacek Dylag"
+  source: "https://unsplash.com/photos/nhCPOp4A2Xo"
+
+---
+
+Our team at [CoverageBook](https://coveragebook.com/) (& [AnswerThePublic](https://answerthepublic.com/)) do _a lot_ with a _very_ small number of developers and designers. Over 4,000 paying customers & 600k monthly vistors (many using a free version of our product) all with a product team of five, including me.
+
+One of the key principles we use to keep our lives calm, and reduce the stress of supporting so much with so few folks, is to buy a lot of software to support our products and reduce the burden on us.
+
+This ends up with _lots_ of paid software to augment our team in critical areas.
+
+We resist building ourselves and pick the simplest, most pleasing-to-use software for the job. We even try, wherever possible not to configure away from the default settings!
+
+Unless I specifically mention it, you can take our usage of these products as an implicit recommendation. Not that there aren’t other alternatives, perhaps better for you, but this is what we’re mostly happy with.
```

Comment:
Second sentence in this paragraph slightly awkward.

Reply:
```suggestion
Unless I specifically mention it, you can take our usage of these products as an implicit recommendation. Although I'd love to hear if you use any great solutions that might improve on what I'm using here.
```